### ISSUE TITLE
Stop note-names from overlapping

### ISSUE THREAD

**Issue by Glitchy-Tozier** [thetwom/Tuner#11]
In those big movements mentioned in #10, tons of note-names overlap.

I suggest only showing octaves of the note you're moving to when the names start getting too close.

![Screenshot_20210628-191315_Tuner.jpg](https://user-images.githubusercontent.com/59611881/123861131-18535b00-d927-11eb-98f7-21a0b47346ab.jpg)

**Comment by thetwom** [comment]
you are right, overlapping notes don't really look professional :-). When I find some time, I will check how this can be improved. Just switching between showing all notes to non-sharp-notes up to octaves must be treated very carefully since it would mean that suddenly notes appear or disappear which can also be confusing.

**Comment by Glitchy-Tozier** [comment]
Agreed, this shouldn't be implemented without thinking of the implications.
